packages/psyche/psyche.0.0.1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+maintainer: "0918nobita"
+synopsis: "A WASM-friendly lightweight programming language implemented in OCaml"
+authors: [
+  "0918nobita"
+]
+homepage: "https://github.com/0918nobita/psyche"
+dev-repo: "git+https://github.com/0918nobita/psyche.git"
+bug-reports: "https://github.com/0918nobita/psyche/issues"
+build: [
+  [make]
+]
+depends: [
+  "ocaml" {>= "4.04.1" & < "4.06.0"}
+  "ocamlfind"
+  "base"
+  "ucorelib"
```

Comment:
```suggestion
```

It does not appear to be used, but maybe I am wrong

Reply:
Right, I'll remove the dependency.